src/Security/MemberAuthenticator/CMSLoginHandler.php

```diff
@@ -1,27 +1,29 @@
 <?php
 
-namespace SilverStripe\Security;
+namespace SilverStripe\Security\MemberAuthenticator;
 
 use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Core\Convert;
+use SilverStripe\Security\CMSSecurity;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
 
-class CMSMemberLoginHandler extends MemberLoginHandler
+class CMSLoginHandler extends LoginHandler
```

Comment:
It's a bit inconsistent between the form being CMSMemberLoginForm. For consistency, one of the two could use a rename.

Haven't done it yet, as it was quite a refactor in multiple places, including ss-admin